packages/compass-query-bar/src/stores/query-bar-store.js

```diff
@@ -1,805 +0,0 @@
- *
- * @returns {Store} The store.
- */
-const configureStore = (options = {}) => {
-  const store = Reflux.createStore({
-    mixins: [StateMixin.store],
-    listenables: options.actions,
-
-    /*
-     * listen to Namespace store and reset if ns changes.
-     */
-    onCollectionChanged(ns, isTimeSeries, isReadonly) {
-      const newState = this.getInitialState();
-      newState.ns = ns;
-      newState.isTimeSeries = isTimeSeries;
-      newState.isReadonly = isReadonly;
-      this.setState(newState);
-    },
-
-    /**
-     * Handle server version updates.
-     *
-     * @param {String} version - The version.
-     */
-    onServerVersionChanged(version) {
-      this.setState({ serverVersion: version });
-    },
-
-    /**
-     * Open the export to language dialog.
-     */
-    exportToLanguage() {
-      this.localAppRegistry.emit('open-query-export-to-language', {
-        filter: this.state.filterString,
-        project: this.state.projectString,
-        sort: this.state.sortString,
-        collation: this.state.collationString,
-        skip: this.state.skipString,
-        limit: this.state.limitString,
-        maxTimeMS: this.state.maxTimeMSString,
-      });
-      this.globalAppRegistry.emit('compass:export-to-language:opened', {
-        source: 'Query',
-      });
-    },
-
-    /**
-     * Initialize the query store.
-     *
-     * @return {Object} the initial store state.
-     */
-    getInitialState() {
-      return {
-        // user-facing query properties
-        filter: DEFAULT_FILTER,
-        project: DEFAULT_PROJECT,
-        sort: DEFAULT_SORT,
-        collation: DEFAULT_COLLATION,
-        skip: DEFAULT_SKIP,
-        limit: DEFAULT_LIMIT,
-        sample: DEFAULT_SAMPLE,
-
-        // internal query properties
-        maxTimeMS: DEFAULT_MAX_TIME_MS,
-
-        // string values for the query bar input fields
-        filterString: '',
-        projectString: '',
-        sortString: '',
-        collationString: '',
-        skipString: '',
-        limitString: '',
-        maxTimeMSString: '',
-
-        // whether Apply or Reset was clicked last
-        queryState: DEFAULT_STATE, // either apply or reset
-
-        // validation flags
-        valid: true,
-        filterValid: true,
-        projectValid: true,
-        sortValid: true,
-        collationValid: true,
-        skipValid: true,
-        limitValid: true,
-        sampleValid: true,
-        maxTimeMSValid: true,
-
-        // last full query (contains user-facing and internal variables above)
-        lastExecutedQuery: null,
-
-        // is the user currently typing (debounced by USER_TYPING_DEBOUNCE_MS)
-        userTyping: false,
-
-        // if the value was populated from a click in the schema view or
-        // query history view.
-        autoPopulated: false,
-
-        // is the query bar component expanded or collapsed?
-        expanded: false,
-
-        // set the namespace
-        ns: '',
```

Comment:
This was used only to be passed to the query-history plugin that already has the namespace information, removed all this logic from this plugin